### ISSUE TITLE
Save currently typed message for connection loss. (Drafts?)

### ISSUE THREAD

**Issue by rixx** [venueless/venueless#133]
On connection loss, the entire application turns to reloading, making you lose your typed input. That can be very frustrating.

**Comment by raphaelm** [comment]
Yeah, I'd love to see that fixed in a way that even survives page reloads and channel switches, i.e. drafts saved to localStorage. drafts saved to user profile is probably overkill
